README.md

```diff
@@ -29,6 +29,41 @@ to the wider research and game developer communities.
 * Visualizing network outputs within the environment
 * Simplified set-up with Docker (Experimental)
 
+## Quick Start Guide
+
+### Get **Unity**
+
+[Download](https://store.unity.com/download) and install Unity, ML-Agents works with versions that are 2017.1 and above. 
+
+### Get Anaconda environment
+
+[Download](https://www.anaconda.com/download) and install Anaconda with Python 3.6 version. 
+
+### Clone the repo and install dependencies
+
+You can get the ML-Agents code and all the packages it depends on with:
+
+    git clone https://github.com/Unity-Technologies/ml-agents.git
+    cd ml-agents/python
+    pip install .
+
```

Comment:
This is not the implicit convention we have for code in markdown : Use ```

Reply:
Will change.